### DISCUSSION TITLE
Welcome to melior Discussions!

### DISCUSSION THREAD

**Discussion by raviqqe** [mlir-rs/melior#265] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jmintb** [comment]
I'll bite. I am a last semester CS master's student experimenting with compiler and language engineering. MLIR looks very interesting so I decided to start building a language with melior and [pest](https://pest.rs/).

Thanks for creating melior!
